src/guide/messages.md

```diff
@@ -0,0 +1,143 @@
+# Signing and verifying messages
+
+Through signing and verifying messages we can provide proof that a digital signature was created by a private key.
+
+## Signing a message
+
+By signing a message using the SDK we can provide a digital signature. Anyone with the message, pubkey and signature can verify the signature was created by the private key of this pubkey.
```

Comment:
Should we format the sign message request payload params: message, pulleys & signature on second part as code?

Reply:
Fixed [fd6678f](https://github.com/breez/breez-sdk-docs/pull/185/commits/fd6678f887ee34c7401dee84db39d55c994d2faa)